app/assets/javascripts/discourse/app/components/header.gjs

```diff
@@ -205,8 +205,8 @@ export default class GlimmerHeader extends Component {
   }
 
   @action
-  toggleUserMenu() {
-    this.header.userVisible = !this.header.userVisible;
+  toggleUserMenu(value) {
```

Comment:
I'm not wild about this variable name, but I can't think of something better. 😄

I don't mind `override`, except that `toggleNavigationMenu()` uses that for a different purpose.

Reply:
Yeah I think this is as good as it gets ATM